docs/PROTOCOL.rst

```diff
@@ -924,17 +954,18 @@ Get a list of features and services supported by the server.
       "hash_function": "sha256"
   }
 
-Protocol Version 1.2
---------------------
-
```

Comment:
Please do not remove this - the structure of the file is currently by version.  At some stage I will switch it to be by method, but that's a big change...